L1Trigger/Phase2L1ParticleFlow/python/L1BJetProducer_cff.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.Phase2L1ParticleFlow.l1pfJetMet_cff import L1TPFJetsExtendedTask
+
+l1tBJetProducerPuppi = cms.EDProducer("L1BJetProducer",
```

Comment:
It would be better to
```suggestion
from L1Trigger.Phase2L1ParticleFlow.L1BJetProducer_cfi import L1BJetProducer as _L1BJetProducer
l1tBJetProducerPuppi = _L1BJetProducer.clone(
    # whatever is changed wrt the default values
)
```
to avoid repeating the defaults, and to make clear what is changed wrt. the defaults.

Reply:
This has been done